docs/tutorials/pretrained_weights.ipynb

```diff
@@ -473,9 +473,6 @@
  ],
  "metadata": {
   "accelerator": "GPU",
-  "colab": {
```

Comment:
I ran https://github.com/kynan/nbstripout to get rid of extra metadata

Reply:
I didn't make as much progress as I had hoped with #2070. If we stick with notebooks, ideally we would use some kind of auto-formatter built into ruff instead of nbstripout. But regardless, we should add a test to CI so there isn't as much back and forth with these files. I would ignore this for now until we make a final decision on how to manage these notebooks.

Reply:
Does this matter?